### ISSUE TITLE
Issue building Inlet_test DiffGrid 

### ISSUE THREAD

**Issue by alknoll** [DOI-USGS/COAWST#1]
Hello,
I am having difficulty compiling the Inlet_test for the DiffGrid case; on first compile, I receive an error with the analytical sediment transport as follows (build.log):

> ana_sediment.h: no values provided for bed, bed_mass, bottom.

If I undefine that part of the problem in the header file, I then get a problem with the current-waves coupling (build2.log):

> master.f90:(.text+0x5d): undefined reference to `__mct_coupler_params_MOD_allocate_coupler_params'
> master.f90:(.text+0x75): undefined reference to `__mct_coupler_params_MOD_nwav_grids'
> master.f90:(.text+0x83): undefined reference to `__mct_coupler_params_MOD_nocn_grids'

I am using the latest version of COAWST (v 3.6).
Build files attached.
[build.log](https://github.com/jcwarner-usgs/COAWST/files/4767997/build.log)
[build2.log](https://github.com/jcwarner-usgs/COAWST/files/4767998/build2.log)

Any help is appreciated. Thanks!
Arie

**Comment by jcwarner-usgs** [comment]
i think both of these issues are related. can you post your coawst.bash file?  is this an updated version?
The ana_sediment in Projects/Inlet_test/DiffGrid is not getting picked up during your build, so i think it has to with the paths listed for my_header_dir and my_ana_dir.

**Comment by alknoll** [comment]
Attached is my coawst.bash file 

I am not sure if it is an updated version, but it came with COAWST v3.6 and I modified it to my needs.

Thanks

[coawst.bash.txt](https://github.com/jcwarner-usgs/COAWST/files/4781572/coawst.bash.txt)

**Comment by alknoll** [comment]
> I was getting a very similar error message when failing to compile Inlet_test Coupled. The issue was that MCT_INCDIR and MCT_LIBDIR were not properly set in my .bashrc file.

Thanks @galenegan but I believe mine is set correctly. 

My paths are
MCT_INCDIR /usr/local/include 
Inside this directory is m_mctworld.mod

MCT_LIBDIR /usr/local/lib
Inside this directory is libmct.a

**Comment by jcwarner-usgs** [comment]
can you check your Compilers/Linux-gfortran.mk (i think this is one you are using)
and see if it has
ifndef USE_SCRIP
             LIBS += $(MCT_PARAMS_DIR)/mct_coupler_params.o
endif

are you using an updated Compilers file?

**Comment by alknoll** [comment]
@jcwarner-usgs It does not have that. It has:
<img width="541" alt="Screen Shot 2020-07-02 at 10 11 49 AM" src="https://user-images.githubusercontent.com/44818547/86369575-a5989c80-bc4c-11ea-94cd-89d05178bede.png">

This is the Compilers file that came with COAWST v3.6.

Thanks,
Arie

**Comment by alknoll** [comment]
@jcwarner-usgs 

I did a clean svn checkout of the code, and now my Linux-gfortran.mk reflects what you have stated (not sure why my previous version was not up to date, since I did the exact same thing on April 30, 2020). I was able to successfully compile COAWST (although I still had the ana_sediment error, I turned off the SEDIMENT process in ROMS), and now it is running. Waiting to see if it completes.

Thank you for your help. It would still be nice to figure out why the SEDIMENT process fails, MY_HEADER_DIR is set correctly, and the ana_sediment.h file is in the Project directory.

Thanks,
Arie
